OpenTabletDriver.Configurations/Configurations/XP-Pen/Artist 22 Pro.json

```
@@ -0,0 +1,41 @@
+{
+  "Name": "XP-Pen Artist 22 Pro",
+  "Specifications": {
+    "Digitizer": {
+      "Width": 476.64,
+      "Height": 268.11,
+      "MaxX": 47663.0,
+      "MaxY": 26809.0
+    },
+    "Pen": {
+      "MaxPressure": 8191,
+      "Buttons": {
+        "ButtonCount": 2
+      }
+    },
+    "AuxiliaryButtons": {
+      "ButtonCount": 0
+    },
+    "MouseButtons": null,
+    "Touch": null
+  },
+  "DigitizerIdentifiers": [
+    {
+      "VendorID": 10429,
+      "ProductID": 2333,
+      "InputReportLength": 64,
+      "OutputReportLength": 0,
+      "ReportParser": "OpenTabletDriver.Configurations.Parsers.XP_Pen.XP_PenReportParser",
+      "FeatureInitReport": null,
+      "OutputInitReport": [
+        "ArAE"
+      ],
+      "DeviceStrings": {},
+      "InitializationStrings": []
+    }
+  ],
+  "AuxilaryDeviceIdentifiers": [],
```

The master branch fixes this typo.
```suggestion
  "AuxiliaryDeviceIdentifiers": [],
```